MC-17386 Update Create Instance Docs (#472)
cloudops/cloudmc-api-docs: source/includes/aws/_instances.md

| `ipPolicies`<br/>*Array*                                | The IP policy to use for the security group if the security group scope is CUSTOM.                                                                           |
| `ipPolicies.ipRange`<br/>*string*                       | The IP range in CIDR block notation of an IPV4 compatible string. Indicates the allowed IP(s) to reach the instance.                                         |
| `ipPolicies.protocol`<br/>*enum*                        | Possible values are UDP, TCP and ICMP. Indicates the allowed protocol to use to reach the instance.                                                          |
| `ipPolicies.portRange`<br/>*integer*                    | The allowed IP range. Can be a single port or a port range, such as 1024-65535                                                                               |
| `blockDeviceMappings.type`<br/>*string*                 | Indicates whether the volume is "Root" or "EBS", used to calculate minimum sizing.                                                                           |

> | `blockDeviceMappings.attachmentDeviceName`<br/>*string* | The device name to set the volume to. Windows and Linux have different ranges to set this to.                                                                |

Reviewer: I think something like this would be better. WDYT ?
```suggestion
| `blockDeviceMappings.attachmentDeviceName`<br/>*string* | The device name for the attached volume. Windows and Linux support different values.                                                                |
```

Reviewer: We can even remove the last sentence and simply keep the generic description if we add a notice

| `blockDeviceMappings.deleteOnTermination`<br/>*boolean* | Indicates behaviour of the volume when the attached instance is deleted.                                                                                     |

> | `blockDeviceMappings.size`<br/>*integer*                | The size of the volume to be set.                                                                                                                            |

Reviewer: ```suggestion
| `blockDeviceMappings.size`<br/>*integer*                | The required size of the attached volume.                                                                                                                            |
```

| `blockDeviceMappings.volumeType`<br/>*string*           | The Amazon EBS volume type (gp2, gp3, io1, io2, st1, sc1, standard). Refer to "Create Volume" docs to see volume types & limits for their respective fields. |

> | `blockDeviceMappings.iops`<br/>*integer*                | Describes the maximum number of input/output operations per second (IOPS) that the volume should provide. Only set for volume types gp3, io1, and io2.       |

Reviewer: ```suggestion
| `blockDeviceMappings.iops`<br/>*integer*                | Describes the maximum number of input/output operations per second (IOPS) supported by the volume type. This field is required for volume types gp3, io1, and io2.       |
```
Correct me if I am wrong @emilyroser

> | `blockDeviceMappings.throughput`<br/>*integer*          | The throughput performance in MiB/s that the volume can support. This field is only valid for gp3 volumes.                                                   |

Reviewer: ```suggestion
| `blockDeviceMappings.throughput`<br/>*integer*          | The throughput performance in MiB/s supported by the volume type. This field is required for gp3 volumes.                                                   |
```

| `sshKey.keyName`<br/>*string*           | The name of the SSH key to be used to connect to this instance via SSH.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |

> | `blockDeviceMappings`<br/>*Array*       | Volumes to create on launch. Leave empty if you want default root volume to be loaded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |

Reviewer 2: ```suggestion
| `blockDeviceMappings`<br/>*Array*       | Volumes to attached to the instance on launch. To be left empty if the default root volume should be used.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

| `ipPolicies.ipRange`<br/>*string*                       | The IP range in CIDR block notation of an IPV4 compatible string. Indicates the allowed IP(s) to reach the instance.                                               |
| `ipPolicies.protocol`<br/>*enum*                        | Possible values are UDP, TCP and ICMP. Indicates the allowed protocol to use to reach the instance.                                                                |
| `ipPolicies.portRange`<br/>*integer*                    | The allowed IP range. Can be a single port or a port range, such as 1024-65535                                                                                     |
| `blockDeviceMappings.type`<br/>*string*                 | Indicates whether the volume is "Root" or "EBS", used to calculate minimum sizing.                                                                                 |
| `blockDeviceMappings.attachmentDeviceName`<br/>*string* | The device name for the attached volume.                                                                                                                           |

> | `blockDeviceMappings.deleteOnTermination`<br/>*boolean* | Indicates behaviour of the volume when the attached instance is deleted.                                                                                           |

Reviewer 2: ```suggestion
| `blockDeviceMappings.deleteOnTermination`<br/>*boolean* | Whether or not the volume should be deleted alongside the instance.                                                                                           |
```

| `blockDeviceMappings.size`<br/>*integer*                | The required size of the attached volume.                                                                                                                          |
| `blockDeviceMappings.volumeType`<br/>*string*           | The Amazon EBS volume type (gp2, gp3, io1, io2, st1, sc1, standard). Refer to "Create Volume" docs to see volume types & limits for their respective fields.       |

> | `blockDeviceMappings.iops`<br/>*integer*                | Describes the maximum number of input/output operations per second (IOPS) supported by the volume type. This field is required for volume types gp3, io1, and io2. |

Reviewer 2: ```suggestion
| `blockDeviceMappings.iops`<br/>*integer*                | Describes the maximum number of input/output operations per second (IOPS) supported by the volume type. This field is only valid for volume types gp3, io1, and io2. |
```

> | `blockDeviceMappings.throughput`<br/>*integer*          | The throughput performance in MiB/s supported by the volume type. This field is required for gp3 volumes.                                                          |

Reviewer 2: ```suggestion
| `blockDeviceMappings.throughput`<br/>*integer*          | The throughput performance in MiB/s supported by the volume type. This field is only valid for gp3 volumes.                                                          |
```

Reviewer 2: can you make the same change in the create volume doc if it's not already done ^
